dis_snek/models/snek/context.py

```diff
@@ -604,5 +604,15 @@ def from_message(cls, client: "Snake", message: "Message"):
     def content_parameters(self) -> str:
         return self.message.content.removeprefix(f"{self.prefix}{self.invoked_name}").strip()
 
+    async def reply(
+        self,
+        content: Optional[str] = None,
+        embeds: Optional[Union[List[Union["Embed", dict]], Union["Embed", dict]]] = None,
+        embed: Optional[Union["Embed", dict]] = None,
```

Comment:
hmm, do you think we should list out all the parameters, to mirror .send()?

Reply:
That's something that's up to another PR, as that would mean `Message.reply` would need to be changed, too. I want to keep this PR rather simple.

It *would* be nice, but it would also be a pain to maintain, so I could argue it either way.